src/mapping/query/functions.rs

```diff
@@ -484,10 +474,13 @@ impl ParseTimestampFn {
         query: Box<dyn Function>,
         default: Option<Value>,
     ) -> Self {
+        let format = Box::new(Literal::from(Value::from(format)));
+        let default = default.map(|v| Box::new(Literal::from(v)) as _);
+
         Self {
             query,
-            format: Box::new(Literal::from(Value::from(format))),
-            default: default.map(|v| Box::new(Literal::from(v)) as _),
+            format,
+            default,
```

Comment:
Unfortunately, that line saving trick only works when there are one or two members of the struct (and no default). It's frustrating that rustfmt won't put 3 or 4 short members on one line, but there it is.

Reply:
Yeah, I agree. I still thought it be more readable to pull them out, just to stay consistent.